contracts/sol/MerkleTree.sol

```diff
@@ -78,8 +78,9 @@ contract MerkleTree is Ownable, Hasher {
     }
 
 
-    // Inserts (appends) a new leaf into the
-    // merkle tree
+    /*
+     * Insert a new leaf
+     */
     function insert(uint256 leaf) public onlyOwner {
         require(nextLeafIndex < maxLeafIndex, "Merkle Tree at max capacity");
```

Comment:
Concerned an attack can fill up the command_tree but probably okay to just make the command tree really big.

But i think this only applies to the deposit and not the message insert.